jujugui/static/gui/src/app/components/deployment/add-credentials/add-credentials.js

```diff
@@ -43,7 +43,7 @@ YUI.add('deployment-add-credentials', function() {
             // controller with work to come to generate a controller name.  Also
             // this key will change from 'state-server' to 'controller'.
             // Makyo 2016-03-30
-            'state-server': 'yellow/aws-eu-central',
+            'state-server': 'jcsackett/amazon',
```

Comment:
I can remove it from the commit, but neither of them are valid, this is going to be removed in the near future in favour of an api call.